modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/CleanupExecutor.java

```diff
@@ -27,346 +27,417 @@
 import java.io.*;
 import java.sql.*;
 import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
-// The main class
+/**
+ * The main class
```

Comment:
Have some meaningful javadocs comments. "The main class" does not give an idea on what this class is about and usage.